### DISCUSSION TITLE
How to preview html files/ enable live preview

### DISCUSSION THREAD

**Discussion by devItGym** [coder/code-server#3088] [Q&A] [answered]
I am reverse-proxying my domain.com -> code-server on 8080 . No preview or live preview extensions work as they all seem to rely on localhost. I also cannot seem to launch browsers via the launch configurations. I tried to install desktop-minimal, and then chromium-browser but to no avail... any ideas/solutions?

thank you!

**Comment by bpmct** [comment, marked as answer]
You can access ports running on localhost in a few ways within code-server. See: https://github.com/cdr/code-server/blob/main/docs/FAQ.md#how-do-i-securely-access-web-services

Easiest solution is by visiting [code-server-url]/proxy/8080

If none of these work for your use case, you can also use something like [ngrok](https://ngrok.com/)
